Analyzer/interface/CommonFunction.h

```diff
@@ -1579,11 +1573,10 @@ reco::DeDxData computedEdx(const int& run_number,
       std::vector<int> amplitudes = convert(cluster->amplitudes());
       std::vector<int> amplitudesPrim = CrossTalkInv(amplitudes,0.10,0.04,true);
 
-     
+      // why is this hardcoded now?
       //if (useClusterCleaning && !clusterCleaning(amplitudes, crossTalkInvAlgo))
```

Comment:
should we keep this commented out version where `crossTalkInvAlgo` is a variable instead of just `1`?

Reply:
Is it sure we do not need `crossTalkInvAlgo` anymore?

Reply:
I kept it, I'm actually saying we should use it :D it's just github doesnt show the line below , this one:
https://github.com/CMS-HSCP/SUSYBSMAnalysis-HSCP/blob/30ca708ec7e6efab6a14b6c3bb24cbaac0e256c4/Analyzer/interface/CommonFunction.h#L1578

so I'm saying why dont we move from
```if (useClusterCleaning && !clusterCleaning(amplitudesPrim, 1))
```
to
```
(useClusterCleaning && !clusterCleaning(amplitudes, crossTalkInvAlgo))
```

Reply:
@dapparu do you agree to do this?

Reply:
Sure I'm ok!